Layers/Renderers/VectorVegetation/PlantMultiMeshes.gd

```diff
@@ -254,8 +244,7 @@ func override_build(center_x, center_y):
 		
 		if is_detailed:
 			mesh_name_to_custom_data[mesh_name].append(Color(
-				mesh_name_to_spritesheet_index[species_mesh_name] \
-						if species_mesh_name in mesh_name_to_spritesheet_index else 0, # Spritesheet index
+				0.0,
```

Comment:
I think it is not very clear what the CUSTOM_DATA does here. Could you maybe explain each "channel"? It appears `r` is obsolete now?